tests/test_cases/test_cocotb/test_deprecated.py

```diff
@@ -0,0 +1,19 @@
+import cocotb
+import warnings
+
+
+@cocotb.test()
+async def test_returnvalue_deprecated(dut):
+    @cocotb.coroutine
+    def get_value():
+        yield cocotb.triggers.Timer(1, units='ns')
+        raise cocotb.result.ReturnValue(42)
+
+    with warnings.catch_warnings(record=True) as w:
+        # Cause all warnings to always be triggered.
+        warnings.simplefilter("always")
+        val = await get_value()
+        assert val == 42
+        assert len(w) == 1
+        assert issubclass(w[-1].category, DeprecationWarning)
+        assert "use return instead" in str(w[-1].message)
```

Comment:
Unindenting these ensures you don't silence warnings you weren't expecting.
```suggestion
    assert val == 42
    assert len(w) == 1
    assert issubclass(w[-1].category, DeprecationWarning)
    assert "use return instead" in str(w[-1].message)
```